src/internal_events/detect_exceptions.rs

```diff
@@ -0,0 +1,11 @@
+use metrics::counter;
+use vector_core::internal_event::InternalEvent;
+
+#[derive(Debug)]
+pub struct DetectExceptionsStaleEventFlushed;
+
+impl InternalEvent for DetectExceptionsStaleEventFlushed {
+    fn emit(self) {
+        counter!("stale_exception_events_flushed_total", 1);
```

Comment:
Do we want to namespace this with something like "detect_exceptions_stale_flushed_total"? Existing internal events are kindof a mixed bag, there's clearly namespaced names like "adaptive_concurrency_limit" for adaptive_concurrency, but there non-namespaced names as well, like "processing_errors_total". Could it be that the namespaced names are for events specific to a component and non-namespaced ones can be triggered by >1 component?

Reply:
updated